examples/game-of-life/assembly/index.ts

```diff
@@ -3,43 +3,49 @@
 // Configuration imported from JS
 import { BGR_ALIVE, BGR_DEAD, BIT_ROT } from "./config";
 
-var w: i32, h: i32, s: i32;
+var width: i32, height: i32, offset: i32;
```

Comment:
Initially, `s` represented `size` here. Both that and `offset` don't seem ideal, any other ideas?